files/image_config/platform/rc.local

```diff
@@ -322,4 +321,13 @@ if [ -f $FIRST_BOOT_FILE ]; then
     firsttime_exit
 fi
 
+# Create dir where following scripts put their output files
+mkdir -p /var/platform
+
+# Write contents of system EEPROM as JSON file
+/usr/bin/decode-syseeprom | /usr/bin/syseeprom-to-json > /var/platform/syseeprom
+
+# Start system information poller
+/usr/bin/syspoll &
```

Comment:
Who would watch & restart, if it crashes ?

Reply:
This is being added as a daemon in https://github.com/Azure/sonic-buildimage/pull/3785 -- so we should remove lines 327-331 from this PR.